llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/base.py

```diff
@@ -248,10 +248,14 @@ def complete(
             request_body=request_body_str,
             max_retries=self.max_retries,
             **all_kwargs,
-        )["body"].read()
-        response = json.loads(response)
+        )
+        response_body = response["body"].read()
+        response_headers = response["ResponseMetadata"]["HTTPHeaders"]
+        response_body = json.loads(response_body)
         return CompletionResponse(
-            text=self._provider.get_text_from_response(response), raw=response
+            text=self._provider.get_text_from_response(response_body),
+            raw=response_body,
+            additional_kwargs=self._get_response_token_counts(response_headers),
```

Comment:
I'm actually not sure if I would put this in additional kwargs? Or at least, the token counting utils in llama-index should be updated to read from there

Reply:
@logan-markewich I was not sure how to get the tokens out. Then I referred to the implementation of llama-index [openai](https://github.com/run-llama/llama_index/blob/a51150c96f956f7c25a77738338579eedcc01798/llama-index-integrations/llms/llama-index-llms-openai/llama_index/llms/openai/base.py#L558) and tried replicating the same.